src/test/java/org/sagebionetworks/EvaluationUtilsTest.java

```diff
@@ -30,10 +39,15 @@ public void testUpdateSubmissionHappyCase() throws Exception {
 		SubmissionStatus actual = new SubmissionStatus();
 		
 		setAnnotation(expected, "foo1", "bar", false);
```

Comment:
This test is not testing anything since it's reusing the internal logic that we are trying to test. It is better to just build an object as expected without reusing the logic that we are testing, can be as simple as setting up the object in this test.